### PR TITLE
add params to the documentation #5

### PR DESCRIPTION
**Pull request by badrinathpatchikolla**

### CODE REVIEW

File: src/main/scala/com/github/music/of/the/ainur/almaren/http/HTTPConnector.scala [hunk 1]
```diff
@@ -45,7 +49,7 @@ private[almaren] case class MainHTTP(
 
     import df.sparkSession.implicits._
 
-    val result = df.mapPartitions(partition => {
+    val result = df.withColumn("""`__URL__`""",urlEncodeing('__URL__)).mapPartitions(partition => {
```
Comment (mantovani) [inline src/main/scala/com/github/music/of/the/ainur/almaren/http/HTTPConnector.scala:52]: What's this ?

Comment (badrinathpatchikolla) [reply src/main/scala/com/github/music/of/the/ainur/almaren/http/HTTPConnector.scala:52]: Encoding the URL field with UDF which is coming from the Dataframe. Maybe I understood wrong sorry for that, if it is wrong please comment out what we want to do in the below issue

```URL
https://github.com/modakanalytics/http.almaren/issues/4
```

Comment (mantovani) [reply src/main/scala/com/github/music/of/the/ainur/almaren/http/HTTPConnector.scala:52]: @badrinathpatchikolla , this should not be default part of the code. You should add the udf and than document it.

Comment (badrinathpatchikolla) [reply src/main/scala/com/github/music/of/the/ainur/almaren/http/HTTPConnector.scala:52]: Sure @mantovani  Will make this as Util and document it in the readMeSteps

Comment (badrinathpatchikolla) [reply src/main/scala/com/github/music/of/the/ainur/almaren/http/HTTPConnector.scala]: Updated in the ReadMe File,Please review it

**Pull request closed without merging**